File: solutions/0167-two-sum-ii---input-array-is-sorted/solution.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& numbers, int target) {
      short left = 0;
      short right = numbers.size() -1;
      short temp;
      while (left != right)
      {
        temp = numbers[left] + numbers[right]; 
        if(temp == target)
        {
            return {left + 1, right + 1};
        }
        else if (temp > target)
            right--;
        else
            left++;
      }
      return {};
    }
};

File: solutions/0167-two-sum-ii---input-array-is-sorted/solution.cs
public class Solution {
    public int[] TwoSum(int[] numbers, int target) {
        int left = 0;
        int right = numbers.Length -1;
        int temp;
        while(left != right)
        {
            temp = numbers[left] + numbers[right];
            if(temp == target)
                return [left + 1,right + 1];
            else if(temp > target)
                right--;
            else left ++;
        }
        return null;
    }
}
